Source/ACE.Server/ACE.Server.csproj

```diff
@@ -10,6 +10,10 @@
     <ServerGarbageCollection>true</ServerGarbageCollection>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="Factories\TestLootGenerationFactory.cs" />
```

Comment:
This should not be modifying the .csproj file.

Reply:
Fixed